go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/minio/minio
 
-go 1.24.0
-
-toolchain go1.24.2
+go 1.24.7
```

Comment:
Please only bump the toolchain. There is nothing to prevent it from compiling on 1.24.0

Reply:
Fixed.